qiskit_bot.yaml

```
@@ -1,30 +1,240 @@
 ---
-# We use quotes around users who don`t want a GitHub notification,
-# but whom we still want their name in the Qiskit Bot message so
-# people know they are relevant.
+always_notify: true
+notification_prelude: |
+  Thanks for contributing to Qiskit documentation!
+
+  Before your PR can be merged, it will first need to pass continuous
+  integration tests and be reviewed. Sometimes the review process can be slow,
+  so please be patient.
```

Abby M wanted an emoji at the end hah. Maybe this?

```suggestion
  so please be patient. Thanks! 🙌
```